deployment/GP02HYPM_Deploy.csv

```diff
@@ -1,99 +1,99 @@
+RB1605,,SR1710,,GP02HYPM-WFP03-03-DOSTAL000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGINS-DOSTAL-01086,50.0796,-144.80572,,4062,4220,
+RB1605,,SR1710,,GP02HYPM-WFP03-04-CTDPFL000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGINS-CTDPFL-00107,50.0796,-144.80572,,4062,4220,
+RB1605,,SR1710,,GP02HYPM-WFP03-05-VEL3DL000,4,1,2016-07-03T02:05:00,2017-07-16T15:20:00,CGMGP-02HYPM-00004,CGVEH-GWFPGX-93602,CGINS-VEL3DL-01024,50.0796,-144.80572,,4062,4220,
+SR1710,,SR1811,,GP02HYPM-MPM01-02-ZPLSGA009,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,,CGINS-ZPLSGA-55110,50.1156,-144.9176,,150,4230,upward-looking
+SR1710,,SR1811,,GP02HYPM-MPM01-02-ZPLSGA010,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,,CGINS-ZPLSGA-55104,50.1156,-144.9176,,150,4230,downward-looking
+SR1710,,SR1811,,GP02HYPM-RIM01-00-SIOENG000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,,CGCON-SIOCON-12371,50.1156,-144.9176,,4073,4230,
+SR1710,,SR1811,,GP02HYPM-RIM01-02-CTDMOG039,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,,CGINS-CTDMOG-10252,50.1156,-144.9176,,160,4230,
+SR1710,,SR1811,,GP02HYPM-WFP02-00-WFPENG000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-99701,CGCON-GWFPGX-99701,50.1156,-144.9176,,2095,4230,
+SR1710,,SR1811,,GP02HYPM-WFP02-01-FLORDL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-99701,CGINS-FLORDL-02934,50.1156,-144.9176,,2095,4230,
+SR1710,,SR1811,,GP02HYPM-WFP02-03-DOSTAL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-99701,CGINS-DOSTAL-01105,50.1156,-144.9176,,2095,4230,
+SR1710,,SR1811,,GP02HYPM-WFP02-04-CTDPFL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-99701,CGINS-CTDPFL-00115,50.1156,-144.9176,,2095,4230,
+SR1710,,SR1811,,GP02HYPM-WFP02-05-VEL3DL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-99701,CGINS-VEL3DL-01026,50.1156,-144.9176,,2095,4230,
+SR1710,,SR1811,,GP02HYPM-WFP03-00-WFPENG000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-10403,CGCON-GWFPGX-10403,50.1156,-144.9176,,4063,4230,
+SR1710,,SR1811,,GP02HYPM-WFP03-01-FLORDL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-10403,CGINS-FLORDL-03284,50.1156,-144.9176,,4063,4230,
+SR1710,,SR1811,,GP02HYPM-WFP03-03-DOSTAL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-10403,CGINS-DOSTAL-01478,50.1156,-144.9176,,4063,4230,
+SR1710,,SR1811,,GP02HYPM-WFP03-04-CTDPFL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-10403,CGINS-CTDPFL-00132,50.1156,-144.9176,,4063,4230,
+SR1710,,SR1811,,GP02HYPM-WFP03-05-VEL3DL000,5,1,2017-07-15T21:25:00,2018-07-26T22:15:00,CGMGP-02HYPM-00005,CGVEH-GWFPGX-10403,CGINS-VEL3DL-01115,50.1156,-144.9176,,4063,4230,

```

Comment:
Deployment 5 looks good.
Single recovery time is correct – mid-water release failed, bottom release valid for all components.